easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.2.eb

```diff
@@ -11,12 +11,15 @@ source_urls = [GNU_SOURCE]
 
 toolchain = {'name': 'GCC', 'version': '4.9.2'}
 
-dependencies = [
-    ('zlib', '1.2.8'),
-]
+dependencies = [('zlib', '1.2.8')]
+
+builddependencies = [('binutils', '2.25', '', True)]
```

Comment:
hmm, not in favour of that solution, we shouldn't force people to install anything on their OS to be able to build stuff with EB, unless there's a very good reason

Can we build the dummy binutils statically, and only include zlib as a build dep there?